### DISCUSSION TITLE
Find command from iOS

### DISCUSSION THREAD

**Discussion by egoitzau** [Z-Hub/Z-Push#47] [General]
Hi!

One little question. The find command issued by iPhones when performing mailbox searchs isn't still supported in Z-push, is it?. I say it because as Microsoft documentation https://learn.microsoft.com/en-us/openspecs/exchange_server_protocols/ms-ascmd/2e6a37ac-1b17-421e-85bc-371cbddde930 it's supported in 16 protocol version but it seems Z-push just arrives until the 14.

Am I wrong?.

Cheers!

**Comment by matidau** [comment]
Hi @egoitzau 

No you are not wrong. Please see this issue for more details https://github.com/Z-Hub/Z-Push/issues/23

Cheers,
Mat
